components/cli/pkg/runtime/runtime.go

```diff
@@ -461,12 +463,37 @@ func WaitFor(checkKnative, hpaEnabled bool) {
 	}
 
 	spinner = util.StartNewSpinner("Checking runtime status (Cellery)...")
-	err = kubectl.WaitForDeployments("cellery-system", time.Minute*15)
+	err = kubectl.WaitForDeployments("cellery-system", waitingTimeCellerySystem())
 	if err != nil {
 		spinner.Stop(false)
 		util.ExitWithErrorMessage("Error while checking runtime status (Cellery)", err)
 	}
 	spinner.SetNewAction("Runtime status (Cellery)...OK")
 	spinner.Stop(true)
+}
 
+func waitingTimeCluster() time.Duration {
+	waitingTime := time.Hour
+	envVar := os.Getenv("CELLERY_CLUSTER_WAIT_TIME")
+	if envVar != "" {
+		wt, err := strconv.Atoi(envVar)
+		if err != nil {
+			util.ExitWithErrorMessage("Error getting waiting time", err)
```

Comment:
Return the error and handle from parent.

Reply:
Updated.